src/main.ts

```diff
@@ -41,16 +42,13 @@ app.get('/.well-known/server-health', (req, res) => {
   res.status(200).send('ok');
 });
 
-// TODO: return appropriate cache headers here
-// let's start with 30 minutes
-// need to research exact header names
-app.get('/', async (req, res) => {
+app.get('/:scheduledSurfaceID/:date', async (req, res) => {
```

Comment:
URL structure here is completely arbitrary, open to suggestions to make it more readable, maybe `/surface/:scheduledSurfaceID/date/:date`?

Reply:
i am all for following conventions